Fix download for v. 3.9

dfev77 opened the pull request:
Also:
 - increase default nunit runner to 2.9.0
 - only take assemblies matching regex /test(s?).dll/

Commit:
Fix download for v. 3.9

Also:
 - increase default nunit runner to 3.9.0
 - upgrade gradle-download-task to fix "Invalid cookie header" error
 - upgrade used gradle version

ngyukman commented on CHANGELOG.md:
```diff
@@ -4,10 +4,15 @@
 * upgrade gradle-download-plugin to 3.2
 * upgrade net.researchgate.release to 2.6.0
 * built with gradle 4.2.1
+* increase default NUnit runner version from 2.6.4 to 3.9.9
```
it has to be in new version, changes before has been released

ngyukman commented on CHANGELOG.md:
```diff
@@ -1,11 +1,23 @@
 # gradle-nunit-plugin changelog
+##1.13
```
miss ###changed?

dfev77 replied:
done

ngyukman commented on CHANGELOG.md:
```diff
@@ -1,11 +1,23 @@
+* built with gradle 4.10.2
+
+### Fixed
+* Apply default parameters on plugin `com.ullink.msbuild` instead of `msbuild`
```
this was an item in 1.12

dfev77 replied:
my bad, fixed

ngyukman commented on src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:
```diff
@@ -86,6 +91,10 @@ class NUnit extends ConventionTask {
             this.metaClass.mixin NUnit3Mixins
         }
+        else
+        {
```
should follow the same code convention, having curly bracket on the previous line

dfev77 replied:
done

ngyukman commented on src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:
```diff
@@ -134,9 +143,13 @@ class NUnit extends ConventionTask {
-            return version.take(version.length() - 2)
+            version = version.take(version.length() - 2)
+            if (isV39)
+            {
```
here as well

dfev77 replied:
done

Commit:
Correct test assembly filtering

Only take assemblies matching regex /test(s?).dll/
Previously all assemblies containing "test" were taken into account.
Now, projects like "XX-test-framework" would not have to do extra
work to set test assemblies, something that would also require to
hardcode the debug/release configuration name in build script.
